kong/db/schema/others/declarative_config.lua

```diff
@@ -37,6 +39,12 @@ local foreign_references = {}
 local foreign_children = {}
 
 
+local function now_updated()
```

Comment:
`now` and `update_time` are only used by `now_updated`, 
so we can minimize the socope with `do end`, like:
```
local now_updated
do
  local now = ngx.now
  local update_time = ngx.update_time
  now_updated = function()
  ...
  end
end

```

Reply:
The style remains consistent with the original code.